docs/docs/reference/release-notes/v5.3/index.md

```diff
@@ -0,0 +1,94 @@
+- [Improved error messages](#improved-error-messages)
+
+Also check out [notable bugfixes](#notable-bugfixes--improvements).
+
+**Bleeding Edge:** Want to try new features as soon as possible? Install `gatsby@next` and let us know if you have any [issues](https://github.com/gatsbyjs/gatsby/issues).
+
+[Previous release notes](/docs/reference/release-notes/v5.2)
+
+[Full changelog][full-changelog]
+
+---
+
+## ES Modules (ESM) in Gatsby files
+
+The ECMAScript module format is the [official TC39 standard](https://tc39.es/ecma262/#sec-modules) for packaging JavaScript. For many years, [CommonJS (CJS)](https://nodejs.org/api/modules.html#modules-commonjs-modules) was the de facto standard in Node.js. Before now, [`gatsby-config`](https://www.gatsbyjs.com/docs/reference/config-files/gatsby-config/) and [`gatsby-node`](https://www.gatsbyjs.com/docs/reference/config-files/gatsby-node/) had to be written in CommonJS (CJS), earlier this year, we also included support for TypeScript, allowing you to write your `gatsby-*` in `.ts`
+
+Support for ESM in `gatsby-config` and `gatsby-node` files has been a highly requested feature in the community, and ecosystems that Gatsby interacts with like mdx and unified have switched to ESM, dropping CJS in their latest major releases. In this release, we've included support for ESM in `gatsby-node.mjs` and `gatsby-config.mjs`. This means that you can now write your `gatsby-*` files in ESM and also use Pure ESM Packages without any hack.
+
+```javascript:title=gatbsy-config.mjs
+//Import Pure ESM or CommonJS packages
+import slugify from '@sindresorhus/slugify';
+
+const config = {
+  siteMetadata: {
+    title: slugify(`Gatsby Default Starter`),
+    siteUrl: `https://www.yourdomain.tld`,
+  },
+  plugins: [],
+};
+
+// Use the export statement
+export default config;
+```
+
+Please note that we do not support ESM in `gatsby-config.ts` and `gatsby-node.ts` yet and support for that would be added in another minor release.
+
+For more details, head over to our [ESM in Gatsby files](https://github.com/gatsbyjs/gatsby/discussions/37069) umbrella discussion.
+
+## Improved error messages
+
+### Show original stack trace from workers
+
+You might have run into an error like this while using Gatsby:
+
+```shell
+ERROR #85928
+
+An error occurred during parallel query running.
+
+Error: Worker exited before finishing task
+```
+
+Gatsby uses [worker threads](https://nodejs.org/docs/latest-v18.x/api/worker_threads.html) to execute its GraphQL queries in parallel (with its own package [`gatsby-worker`](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-worker)). When Parallel Query Running was introduced these workers were already instructed to propagate their errors to the main process so that you can see what actually happened. Unfortunately, at a certain workload the workers didn't have enough time to relay those messages and you were left alone with the above mentioned worker error.
```

Comment:
```suggestion
Gatsby uses [child processes](https://nodejs.org/docs/latest-v18.x/api/child_process.html) to execute its GraphQL queries in parallel (with its own package [`gatsby-worker`](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-worker)). When Parallel Query Running was introduced these workers were already instructed to propagate their errors to the main process so that you can see what actually happened. Unfortunately, at a certain workload the workers didn't have enough time to relay those messages and you were left alone with the above mentioned worker error.
```

Reply:
There's one more mention of "workers" below. Do we need to change that too?

Reply:
Ah yes, I edited my suggestion. I kept "worker" in "worker error" sentence as that makes sense because it mentions "Error: Worker exited before finishing task"